pygeoapi/api.py

```diff
@@ -1045,9 +1045,7 @@ def describe_collections(self, request: Union[APIRequest, Any],
             if 'format' in collection_data:
                 collection_data_format = collection_data['format']
 
-            is_vector_tile = (collection_data_type == 'tile' and
-                              collection_data_format['name'] not
-                              in [F_PNG, F_JPEG])
+            is_raster_tile = collection_data['name'] == 'WMTSFacade'
```

Comment:
I do not like the check this way, but at this point in the function we do not have the actual provider object, but only the configuration.

I see 2 ways to resolve this:
* We can include the tile type into the configuration. To be consistent this should be done for vector tiles as well.
* We can move the link generation inside the function to appear after the instantiation of the providers.

Reply:
A third way is to test for the format, but the check might need to be updated if at any time another format is introduced.

Reply:
> I do not like the check this way, but at this point in the function we do not have the actual provider object, but only the configuration.
> 
> I see 2 ways to resolve this:
> 
> * We can include the tile type into the configuration. To be consistent this should be done for vector tiles as well.
> * We can move the link generation inside the function to appear after the instantiation of the providers.

Including the tile type into the configuration would face the same problem that I described above. However, I think it is worth to explore your second option.